ls1intum/Artemis
# Use JDK16 for Java programming exercises

Update the Java Programming Exercise Template to JDK16

### Checklist

### Steps for Testing
1. Create a programming exercise (with and without SCA)
2. Participate in it and check if everything is working correctly
